Code/Framework/AzCore/AzCore/Serialization/std/VariantReflection.inl

```diff
@@ -505,9 +503,7 @@ namespace AZ
                 };
 
                 AZStd::visit(AZStd::move(alternativeVisitor), *reinterpret_cast<const VariantType*>(variantPtr));
-                // To avoid including ObjectStream.h into this file, we static cast the value of 0
-                // to an AZ::ObjectStreamWriteElemntResponse which corresponds to the CompletedWrite enum value
-                return static_cast<AZ::ObjectStreamWriteOverrideResponse>(0);
+                return AZ::ObjectStreamWriteOverrideResponse::CompletedWrite;
```

Comment:
This file is always (and only) included with Serialization.h and this definition for the enum will always be available.